examples/decomposition/plot_faces_decomposition.py

```diff
@@ -80,6 +80,7 @@ def plot_gallery(title, images, n_col=n_col, n_row=n_row, cmap=plt.cm.gray):
     ('Sparse comp. - MiniBatchSparsePCA',
      decomposition.MiniBatchSparsePCA(n_components=n_components, alpha=0.8,
                                       n_iter=100, batch_size=3,
+                                      normalize_components=True,
```

Comment:
You should remove the argument because we will get another deprecation warning :)

Reply:
If i remove the parameter `normalize_components` the default `False` arguments returns the errors. What should i do then?